src/modules/access/access.service.ts

```
@@ -128,250 +60,141 @@ type Role = 'admin' | 'underboss' | 'caporegime' | 'host' | null;
-    const countryAccesses = await this.knexService
-      .knex<ICountryAccess>('country_access')
-      .where('user_telegram_id', telegram_id);
+    const countryAccess = await this.getCountryAccess(telegram_id);
 
-    for (const countryAccess of countryAccesses) {
-      interface ICountry {
-        id: string;
-        name: string;
-      }
+    if (countryAccess.length) {
+      const countries = await this.countryService.getCountriesByCountryIds(countryAccess.map((access) => access.id))
```

ESLint

[error] Replace `countryAccess.map((access)·=>·access.id))` with `⏎········countryAccess.map((access)·=>·access.id),⏎······);`

(prettier/prettier)